server/odc-service/src/main/java/com/oceanbase/odc/service/task/util/HttpUtil.java

```diff
@@ -92,4 +99,40 @@ public static <T> T request(String method, String uri, Map<String, String> heade
         return JsonUtils.fromJson(response, responseTypeRef);
     }
 
+    public static boolean isConnectable(String server, int servPort, int retryTimes) {
+        boolean connectable = false;
+        for (int i = 0; i < retryTimes; i++) {
+            connectable = isConnectable(server, servPort);
+            if (connectable) {
+                break;
+            }
+        }
+        return connectable;
+    }
+
+    public static boolean isConnectable(String server, int servPort) {
+        try {
+            InetAddress ad = InetAddress.getByName(server);
+            boolean state = ad.isReachable(1000);
```

Comment:
is 1 second too short?

Reply:
add odc server healthy check in HttpUtil